src/c++/perf_analyzer/genai-perf/tests/test_llm_metrics.py

```diff
@@ -169,38 +134,38 @@ def test_triton_llm_profile_data(self, mock_read_write: pytest.MonkeyPatch) -> N
 
         # Disable Pylance warnings for dynamically set attributes due to Statistics
         # not having strict attributes listed.
-        assert stat.avg_time_to_first_token == 2  # type: ignore
-        assert stat.avg_inter_token_latency == 2  # type: ignore
+        assert stat.avg_time_to_first_token == 2e-6  # type: ignore
```

Comment:
I haven't looked too closely into this code, but I would not expect these unit tests to need to change. Are you scaling the actual source stat data? Or just scaling the output?

Reply:
This still feels wrong to me, but if it is clearly documented what units the stats are in then it could be ok. My opinion is still that we keep everything in raw form and only scale when printed

Reply:
Moved the scaling to the output reporter.
This feels like the right compromise to me. Duplicating this work in every exporter seems incorrect but scaling the stats is probably not correct either.